tools/functional-tester/etcd-runner/command/lease_renewer_command.go

```diff
@@ -54,7 +53,7 @@ func runLeaseRenewerFunc(cmd *cobra.Command, args []string) {
 			err error
 		)
 		for {
-			l, err = c.Lease.Grant(ctx, 5)
+			l, err = c.Lease.Grant(ctx, 30)
```

Comment:
Since sigstop might stop runner for a long time in the case of compaction, lease will be expired if it's 5 seconds. I do agree a command line argument is better.